core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -135,6 +140,35 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
         });
     };
 
+    $scope.edit = function() {
+      SkillBackendApiService.fetchSkillAsync(
+        suggestion.change.skill_id).then((skillDict) => {
+        var skill = skillDict.skill;
```

Comment:
Prefer `const`. Try to avoid using `var` unless working with a legacy file using exclusively `var`.